Projects/Foundations/Schemas/Docs-Nodes/E/Exchange/Exchange-Learning-Products/exchange-learning-products.json

```diff
@@ -9,18 +19,14 @@
                     "language": "RU",
                     "text": "Щелкните правой кнопкой мыши и Edit, чтобы войти в режим редактирования и изменить этот текст. ENTER для написания новых абзацев. ESC для выхода из режима редактирования. Затем вы можете использовать docs.save для сохранения изменений и app.contribute для отправки их в основной репозиторий.",
                     "updated": 1645166381126
+                },
+                {
+                    "language": "TR",
+                    "text": "Düzenleme moduna girmek ve bu metni değiştirmek için sağ tıklayın ve Düzenle deyin. Yeni paragraflar yazmak için ENTER tuşuna basın. Düzenleme modundan çıkmak için ESC. Daha sonra değişikliklerinizi kaydetmek için docs.save ve ana depoya göndermek için app.contribute kullanabilirsiniz.",
+                    "updated": 1654392334416,
+                    "style": "Text"
                 }
             ]
         }
-    ],
-    "definition": {
-        "text": "Right click and select the pencil button to edit this tex. Replace it with a definition / summary. Hit ESC to exit edit mode.",
-        "translations": [
-            {
-                "language": "RU",
-                "text": "Щелкните правой кнопкой мыши и выберите кнопку с карандашом, чтобы отредактировать этот текст. Замените его на определение / резюме. Нажмите ESC, чтобы выйти из режима редактирования.",
-                "updated": 1645166371797
-            }
-        ]
-    }
-}
```

Comment:
I guess same thing here? Just moved to the correct position?

Reply:
Yes. "type": Moved under "Exchange Learning Products". Where it should be.